examples/with-turso/app/form.tsx

```diff
@@ -0,0 +1,47 @@
+"use client";
+
+import { useRef } from "react";
+import { useFormStatus } from "react-dom";
+
+import { addTodo } from "./actions";
+
+export function Form() {
+  const formRef = useRef<HTMLFormElement>(null);
+
+  formRef.current?.reset();
+
+  return (
+    <form
+      action={async (formData) => {
+        await addTodo(formData);
+        formRef.current?.reset();
+      }}
+      className="rounded-md border border-gray-300 p-3 shadow-sm"
+      ref={formRef}
+    >
+
+      <input
+        id="description"
+        name="description"
+        placeholder="Insert new todo"
+        className="w-full text-black outline-none"
+        required
+        aria-label="Description of todo"
+        type="text"
+        autoFocus
+      />
+
+      <Submit />
+    </form>
+  );
+}
+
+export function Submit() {
```

Comment:
I don't see `<Submit>` used anywhere else. Should we remove the `export` and maybe even inline the button into the form code above?

Reply:
Removed export. Is it possible to invoke `useFormStatus` in the same component as the form now? I seen some comments from the community about this not being possible.